src/tools/rest-frontend/CMakeLists.txt

```diff
@@ -42,7 +42,7 @@ else ()
     # install files to destination
     install (DIRECTORY public resources DESTINATION ${install_directory})
     install (FILES package.json README.md ${CMAKE_CURRENT_BINARY_DIR}/Gruntfile.js DESTINATION ${install_directory})
-    if (NOT EXISTS ${install_directory}/application-config.json)
+    if (NOT EXISTS ${CMAKE_INSTALL_PREFIX}/${install_directory}/application-config.json)
```

Comment:
Then I'll change the mount script to:
- ensure a copy of the application-config.json exists (default > custom)
- mounts the copy of the config file
- and add a hint in the readme of course